tests/Collection/CollectionFunctionalTest.php

```diff
@@ -311,6 +311,71 @@ public function testMapReduce()
         $this->assertNotEmpty($result->getCounts());
     }
 
+    public function testDontInheritWriteConcernInTransaction()
+    {
+        $this->skipIfTransactionsAreNotSupported();
+
+        $this->createCollection();
+
+        $session = $this->manager->startSession();
+        $session->startTransaction([
+            'readConcern' => new \MongoDB\Driver\ReadConcern("snapshot"),
+            'writeConcern' => new \MongoDB\Driver\WriteConcern(\MongoDB\Driver\WriteConcern::MAJORITY)
+        ]);
+
+        $collection = new Collection(
```

Comment:
Assuming you're going to do this frequently in these tests, this may be more concise:

```
$collection = $this->collection->withOptions([
    'writeConcern' => new \MongoDB\Driver\WriteConcern(1),
]);
```